src/connect/component.jsx

```diff
@@ -0,0 +1,80 @@
+/* @flow */
+import { loadAxo } from "@paypal/connect-loader-component";
+import { stringifyError } from "@krakenjs/belter/src";
+
+import {
+  getClientID,
+  getClientMetadataID,
+  getUserIDToken,
+  getLogger,
+} from "@paypal/sdk-client/src";
+
+import { sendCountMetric } from "./sendCountMetric";
+
+// $FlowFixMe
+export const getConnectComponent = async (merchantProps) => {
+  sendCountMetric({
+    name: "pp.app.paypal_sdk.connect.init.count",
+    dimensions: {},
+  });
+
+  const cmid = getClientMetadataID();
+  const clientID = getClientID();
+  const userIdToken = getUserIDToken();
+  const { metadata } = merchantProps;
+
+  let loadResult = {};
+  try {
+    loadResult = await loadAxo({
+      platform: "PPCP",
+      btSdkVersion: "3.97.3-connect-alpha.6.1",
+      minified: false,
```

Comment:
Should this be true for when we merge it? Assuming minified was for testing locally

Reply:
Good eye. We have a ticket to map the `debug` query param to this, but you're right that we should probably set it to `true`